backend/routers/setApps.py

```diff
@@ -0,0 +1,25 @@
+from fastapi import APIRouter, HTTPException, Request,Depends
+from utils.commonFunction import validateJwtFromCookies
+from core.db import getDatabase
+
+
+
+db = getDatabase()
+router = APIRouter(
+    prefix="/setApps",
+    tags=["setApps"]
+)
+
+@router.put("/setAppsToUser")
+async def addApps(apps: str, request: Request,token= Depends(validateJwtFromCookies)):
+    token = token.get("Email")
+  
+    # Check if a user with the same Email already exists
+    existing_user = await db["User"].find_one({"Email": token})
+
+    if not existing_user:
+        raise HTTPException(
+            status_code=400, detail="User with the same Email does not exist")
+    # Insert the apps for the user
+    await db["user"].update_one({"Email": token}, {"$set": {"apps": apps}})
+    return {"detail": "Apps updated successfully"}
```

Comment:
Try to have these APIs in a try catch in case something goes wrong return the error message. If it just fails, FastAPI will show `Internal Server Error` which does not help with debugging.